migration/src/jira2github_import.py

```diff
@@ -93,6 +102,10 @@ def convert_issue(num: int, dump_dir: Path, output_dir: Path, account_map: dict[
 
         try:
             body = f'{convert_text(description, att_replace_map, account_map, jira_users)}\n\n'
+            for image_file in unmentioned_images:
+                # show orphaned (unmentioned) image files in the issue description
+                att_url = att_replace_map.get(image_file)
+                body += f'![{image_file}]({att_url})\n\n'
```

Comment:
Should you put a leading double `\n` instead of trailing?  This way the image won't glomm onto the end of the body text?

Reply:
I think it's fine since two trailing line feeds were already added at the end of the `body` string. 
https://github.com/apache/lucene-jira-archive/blob/c4c28c273302140626785ba2ecb106f1b3ccedbf/migration/src/jira2github_import.py#L95

Reply:
Ahh super, thanks!